SECURITY.md

```diff
@@ -20,8 +20,96 @@ Container also can import certificates from Kubernetes.
 If `SEC_IMPORT_K8S_CERTS` is set to `true` and `/var/run/secrets/kubernetes.io/serviceaccount` folder is mounted into container the `.crt` files will be imported into the the truststore file. Default value is `false`.
 
 
-### Providing custom keystore
+### Providing a custom keystore
 
 A custom keystore can be provided during the application image's build phase by simply copying the keystore into the image's  `/output/resources/security/key.p12` location. 
 
 You must then override the keystore's password by including your copy of the `keystore.xml` file inside the `/config/configDropins/defaults/` directory.
+
+## Single Sign-On configuration
```

Comment:
Please add (`20.0.0.5`) to indicate when this support was added to WebSphere Liberty images.

Also, enhance the description for the Security section in the main page: similar to this for OL - https://github.com/OpenLiberty/ci.docker#security

Reply:
There is another subsection called **Security** under top level section **OpenJ9 Shared Class Cache (SCC)**. I added a top level section called **Security** same as the OL repo. Hopefully it's not confusing.